packages/api-reference/src/features/ExternalDocs/ExternalDocs.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+import { ScalarMarkdown } from '@scalar/components'
+import { type OpenAPIV3_1 } from '@scalar/openapi-types'
+
+defineProps<{
+  value?: OpenAPIV3_1.ExternalDocumentationObject
+}>()
+</script>
+
+<template>
+  <template v-if="value">
+    <a
+      :href="value.url"
+      target="_blank"
+      rel="noopener noreferrer">
+      <ScalarMarkdown
+        v-if="value.description"
+        :value="value.description" />
+      <template v-else>
+        {{ value.url }}
+      </template>
```

Comment:
We need to make sure this looks good with long content (is clipped or something).